sdcm/cluster.py

```diff
@@ -2768,12 +2768,18 @@ def _node_setup(self, node, seed_address):
         with open(yaml_dst_path, 'r') as f:
             scylla_yaml_contents = f.read()
 
-        node_private_ip_address = node.private_ip_address
-        if not self.seed_nodes_private_ips:
-            self.seed_nodes_private_ips = [node_private_ip_address]
-            seed_address = node_private_ip_address
-        else:
-            seed_address = ','.join(self.seed_nodes_private_ips)
+        # Fixme: there is some code that assume the first node as seed,
+        # so we can't choose the fast one as seed, let's disable the code.
+        # The gce instances are created in parallel right now, this
+        # optimization is also not necessary
+        # https://github.com/scylladb/scylla-cluster-tests/issues/241
+        #
+        # node_private_ip_address = node.private_ip_address
+        # if not self.seed_nodes_private_ips:
+        #     self.seed_nodes_private_ips = [node_private_ip_address]
+        #     seed_address = node_private_ip_address
+        # else:
+        #     seed_address = ','.join(self.seed_nodes_private_ips)
```

Comment:
Yes, since you have fixed the GCE code to use parallel instance creation, it makes sense to simply pick the first node and use it as the seed node.